src/sinks/humio/logs.rs

```diff
@@ -25,37 +25,98 @@ use crate::{
 
 const HOST: &str = "https://cloud.humio.com";
 
-#[derive(Clone, Debug, Deserialize, Serialize)]
+/// Configuration for the `humio_logs` sink.
+#[configurable_component(sink)]
+#[derive(Clone, Debug)]
 #[serde(deny_unknown_fields)]
 pub struct HumioLogsConfig {
+    /// The Humio ingestion token.
     pub(super) token: String,
-    // Deprecated name
+
+    /// The base URL of the Humio instance.
     #[serde(alias = "host")]
     pub(super) endpoint: Option<String>,
+
+    /// The source of events sent to this sink.
```

Comment:
I don't have a suggestion, but I'm wondering if this could be confusing with Vector's internal use of "source" as well.

Reply:
Hmmm, fair point, but yeah, not sure what a better phrase would be given that the field in Humio is literally called `@source`.

Reply:
Yep 🤷 not a good suggestion or anything to change - I guess there was an old suggestion that we'd be renaming sources/sinks but yeah. Fine as is. Naming is hard.